packages/coin98-wallet/package.json

```diff
@@ -1,7 +1,27 @@
 {
   "name": "@near-wallet-selector/coin98-wallet",
   "version": "7.4.0",
+  "description": "Coin 98 wallet package for NEAR Wallet Selector.",
+  "keywords": [
+    "near",
+    "blockchain",
+    "wallets",
+    "dapps",
+    "near-protocol",
+    "near-blockchain",
+    "wallet selector",
+    "injected wallet",
+    "coin 98"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/near/wallet-selector/tree/main/packages/coin98-wallet"
+  },
+  "bugs": {
+    "url": "https://github.com/near/wallet-selector/issues"
+  },
+  "homepage": "https://www.npmjs.com/package/@near-wallet-selector/coin98-wallet",
```

Comment:
For `homepage` I would set it to the folder/location of the package in GitHub `https://github.com/near/wallet-selector/tree/main/packages/coin98-wallet` since in NPM if we click the link as it is right now it will just refresh the NPM page.